SnapMD.ConnectedCare.ApiModels/PatientConsultationInfo.cs

```diff
@@ -24,6 +24,11 @@ public class PatientConsultationInfo
         public string AssignedDoctorFirstName { get; set; }
         public string AssignedDoctorLastName { get; set; }
 
+        /// <summary>
+        /// For patients who depend from they guardians this flag shows whether patient authorized by any of his guardians or not.
+        /// </summary>
+        public bool IsAuthorized { get; set; }
```

Comment:
@lorddev I will remove comment if it confused. 
This property necessary for storing value from PatientProfile.IsAuthorized() method for more details see SnapMD.Data.Entities.PatientProfile class. 

This information necessary in Consultation page.